mcs/class/corlib/Test/System/NumberFormatterTest.cs

```diff
@@ -1567,6 +1568,7 @@ public void Test05028 ()
 		}
 
 		[Test]
+		[Category("NotWorking")] // bug in .NET Core: https://github.com/dotnet/coreclr/issues/18218
```

Comment:
seems like a fix was already merged: https://github.com/dotnet/coreclr/pull/18221, can we cherrypick this into our corefx fork and reenable the test?

Reply:
Sure!